### PR TITLE
Feature/openid Added google api auth

### PR DESCRIPTION
**Pull request by MaxSukhovetskyi**
I create app from my account do not change secret or client_id

### CODE REVIEW

File: easyrest/views/auth_controler.py [hunk 1]
```diff
@@ -59,6 +61,65 @@ def check_token(request):
...
+    if not email or not new_pass:
+        raise HTTPBadRequest("no email in token")
+
+    user = request.dbsession.query(User).filter_by(email=email).first()
+    # check if user exists    
+    if user is None:
+        form_data = {
+            "email": email,
+            "password": new_pass,
+            "name": decoded_token.get('name', ''),
```
Comment (RazSKg) [inline easyrest/views/auth_controler.py]: You done this above and create variable. You cat use it here

File: easyrest/views/auth_controler.py [hunk 2]
```diff
@@ -59,6 +61,65 @@ def check_token(request):
...
+            "img": decoded_token.get('picture', '')
```
Comment (RazSKg) [inline easyrest/views/auth_controler.py]: As above

File: frontend/src/components/Login.jsx [hunk 3]
```diff
@@ -179,6 +179,12 @@ class Login extends React.Component {
               </Snackbar>
               <Grid item xs={12}>
                 <Grid container justify="space-between">
+                  <Button
+                    href="https://accounts.google.com/o/oauth2/v2/auth?client_id=284454521138-k70hk283vmsdtj8m3gh242gmudk8q3k3.apps.googleusercontent.com&response_type=id_token&nonce=123&redirect_uri=http://localhost:3000/openid&scope=email+profile"
```
Comment (RazSKg) [inline frontend/src/components/Login.jsx:183]: 😊

**Review by RazSKg — APPROVED** [review]
Nice, I have some questions, but will ask them later. 😊

File: easyrest/views/auth_controler.py (hunk 1 above)
Comment (MaxSukhovetskyi) [reply easyrest/views/auth_controler.py]: Lol, didn't see that

**Pull request merged**